### PR TITLE
 simple notification to stdout. Useful for testing 

### PR DESCRIPTION
**Pull request by december-soul**
This will just print out the message to stdout.
Useful for testing

### CODE REVIEW

File: app/defaults.yml [hunk 1]
```diff
@@ -52,6 +52,13 @@ notifiers:
     optional:
       username: null
       password: null
+  stdout:
+    required:
+      enable: null
+    optional:
+      parse_mode: html
```
Comment (shads2) [inline app/defaults.yml]: Is `parse_mode: html` required for this?

Comment (december-soul) [reply app/defaults.yml]: not, 
parse_mode: html
will not be used

Comment (shads2) [reply app/defaults.yml]: If you can make that change and then I can merge this PR.

**Comment by december-soul** [comment]
I hope it fits now

**Comment by shads2** [comment]
Looks good to me 👍

**Pull request merged**